Automattic/Simplecons
# Add unpin icon

### Fix
- Add unpin icon

<img width="291" alt="Screenshot 2020-09-02 at 15 19 54" src="https://user-images.githubusercontent.com/36532468/91995499-cd30e180-ed2f-11ea-9014-9c034ccceda3.png">

### To test
- Verify that the icon appears as intended on a 24x24px pixel grid
- Verify that the paths match those of the existing icon `pinned`
